src/interfaces/issues-processor-options.ts

```diff
@@ -47,4 +47,6 @@ export interface IIssuesProcessorOptions {
   exemptAllIssueAssignees: boolean | undefined;
   exemptAllPrAssignees: boolean | undefined;
   enableStatistics: boolean;
+  removeLabelsWhenUnstale: string;
```

Comment:
I don't want to be too picky on naming, but I wonder if a name like `labelsToRemoveWhenUnstale` would read a little better and indicate that this option is a list of labels. To me `removeLabelsWhenUnstale` reads like it should be a `boolean` option. Not a huge deal either way, but curious what your thoughts are on that? @BenVillalobos @C0ZEN

Reply:
> reads like it should be a boolean option

I agree, and that makes sense considering I followed the naming scheme from `removeIssueStaleWhenUpdated` 😄 will rename.